bitrise.yml

```diff
@@ -33,31 +29,47 @@ workflows:
       - script@1:
           inputs:
             - content: ./gradlew ktlint detekt lintRelease apiCheck
-  test:
-    before_run:
-      - prepare_all
-    after_run:
-      - conclude_all
+  conclude_all:
     steps:
-      - script@1:
-          title: Restore gradle.properties
+      - script-runner@0:
+          is_always_run: true
+          title: Copy test results to tmp
           inputs:
-            - content: mkdir -p ~/.gradle ; cp .bitrise/ci-gradle.properties ~/.gradle/gradle.properties
-      - script@1:
+            - file_path: ./scripts/copy_test_results_to_tmp.sh
+      - deploy-to-bitrise-io@2:
           inputs:
-            - content: ./gradlew testDebugUnitTest verifyPaparazziDebug -x :stripe-test-e2e:testDebugUnitTest
+            - notify_user_groups: none
+            - is_compress: "true"
+            - deploy_path: /tmp/test_results
+            - is_enable_public_page: "false"
+          title: Deploy test results artifacts
+      - cache-push@2: {}
   maestro-financial-connections:
     before_run:
       - prepare_all
     after_run:
       - conclude_all
     steps:
-      - avd-manager@1: { }
-      - wait-for-android-emulator@1: { }
+      - avd-manager@1: {}
+      - wait-for-android-emulator@1: {}
       - script-runner@0:
           title: Execute Maestro tests
           inputs:
             - file_path: ./scripts/execute_maestro_tests.sh
+      - script@1:
+          inputs:
+            - content: |
+                #!/usr/bin/env bash
+                set -e
+                set -o pipefail
+                set -x
+                
+                ruby ./scripts/notify_failure_endpoint.rb \
+                $SDK_FAILURE_NOTIFICATION_ENDPOINT \
+                $SDK_FAILURE_NOTIFICATION_ENDPOINT_HMAC_KEY \
+                $BITRISE_BUILD_URL \
+                RUN_BANKCON_MOBILE
+          title: Notify failure endpoint
```

Comment:
This is the only new content on the file. Exported from bitrise editor, we can revert the other changes but it feels right keeping it in sync with bitrise export format?